pkg/consts/consts.go

```diff
@@ -18,4 +18,6 @@ const (
 	EtcdOperatorNamespace = "storageos-etcd"
 
 	EtcdSecretName = "storageos-etcd-secret"
+
+	PortalManagerLatestVersion = "v1.0.5"
```

Comment:
On this way i guess we have to create a new release each time we update PM. This should be a feature and a problem too :D WDYT? Portal manager release cycle is different than plugin's

Reply:
I'm hoping this will be a temporary solution until we can make portal-manager public, then there's no need for any version pinning!
But until that happens, I think this is probably a better option than deploying the `develop` portal manager which is a bit scary.